crates/deltalake-core/src/operations/transaction/state.rs

```diff
@@ -296,6 +297,12 @@ impl<'a> PruningStatistics for AddContainer<'a> {
         });
         ScalarValue::iter_to_array(values).ok()
     }
+
+    // This function is required since DataFusion 35.0, but is implemented as a no-op
```

Comment:
This function was introduced in DataFusion 35, but is implemented as always returning None by DataFusion, so I matched the behavior